apps/src/codebridge/FilePreview/HTMLPreview.tsx

```diff
@@ -22,14 +22,7 @@ import moduleStyles from './styles/html-preview.module.scss';
 const SOURCE_CHANGE_DELAY_MS = 500;
 
 export const HTMLPreview: React.FC = () => {
-  const pageAction = getLabViewPageAction();
   const isFullScreenView = useAppSelector(state => state.lab.isFullScreenView);
-  const iframeHeightClass = useMemo(() => {
-    if (pageAction === 'share' || isFullScreenView) {
```

Comment:
Does it still work as expected in share view/full screen? (edit: I see in the video it works in full screen, so just wondering about share)

Reply:
Yep! I just checked Share and it seems to work (tested on mobile to get a scroll):


https://github.com/user-attachments/assets/f2e4f076-52f2-4fb8-bf82-b490135340f6